src/views/List.jsx

```diff
@@ -1,17 +1,43 @@
+import { useState } from 'react';
 import { ListItem } from '../components';
 
 export function List({ data }) {
+	const [filter, setFilter] = useState('');
+
+	const clickHandler = (event) => {
+		event.preventDefault();
+		setFilter('');
+	};
+
 	return (
 		<>
-			<p>
-				Hello from the <code>/list</code> page!
-			</p>
+			<form>
+				<label htmlFor="itemInput">Filter Items: </label>
+				<input
+					type="text"
+					id="itemInput"
+					name="itemInput"
+					placeholder="Start typing here..."
+					value={filter}
+					onChange={(event) => setFilter(event.target.value)}
+				/>
+				{filter && (
+					<span>
+						<button onClick={clickHandler}>X</button>
+					</span>
+				)}
+			</form>
 			<ul>
 				{data.length > 0 ? (
 					<ul>
```

Comment:
I think this came from the previous versions of this component but I never noticed it until now. Should the inner elements being created by the map be `<li>` tags rather than `<ul>` tags?

Reply:
Ah yes! Good catch. I have fixed that and also added a type="reset" to the button. If that is not correct and it should just be type="button" then let me know.

Reply:
Awesome, all looks good to me. 👍